in_toto/in_toto_sign.py

```diff
@@ -84,25 +94,47 @@ def _sign_and_dump_metadata(metadata, args):
     if not args.append:
       metadata.signatures = []
 
-    for key_path in args.key:
-      key = util.prompt_import_rsa_key_from_file(key_path)
-      metadata.sign(key)
-
-      # Only relevant when signing Link metadata, where there is only one key
-      keyid = key["keyid"]
-
+    signature = None
+    # If the cli tool was called with `--gpg [KEYID ...]` `args.gpg` is
+    # a list (not None) and we will try to sign with gpg.
+    # If `--gpg-home` was not set, args.gpg_home is None and the signer tries
```

Comment:
Just making sure: as of now it also respects if the `GNUPG_HOME` env_var is set before running

Reply:
yep that works